fem/datacollection.hpp

```diff
@@ -54,16 +65,16 @@ class DataCollection
    /// The (common) mesh for the collected fields
    Mesh *mesh;
 
-   /** Time cycle; for time-dependent simulations cycle >= 0, otherwise = -1.
-       When cycle >= 0, it is appended to directory names. */
+   /// Time cycle; for time-dependent simulations cycle >= 0, otherwise = -1.
+   /**  When cycle >= 0, it is appended to directory names. */
    int cycle;
    /// Physical time (for time-dependent simulations)
    double time;
 
    /// Time step i.e. delta_t (for time-dependent simulations)
    double time_step;
 
-   /// Serial or parallel run?
+   /// Serial or parallel run? True iff mesh is a ParMesh
    bool serial;
```

Comment:
Can this parameter be different than the ``format``?  
E.g. is ``serial`` independent of the ``format``.  

I saw a comment above that serial meshes are not supported when ``format == PARALLEL_FORMAT``.

Reply:
There is a typo here - the comment should say "False iff mesh is a ParMesh".

Anyway, to answer your question: if `serial` is false, both formats can be used, so right now the decision about which format to actually use (when saving data) is based on both `serial` and `format`. When reading data, `format` should be sufficient and `serial` is set based on the data that is read.